pkg/apis/operator/v1alpha1/common.go

```diff
@@ -59,6 +59,8 @@ type KComponentSpec interface {
 	GetResources() []ResourceRequirementsOverride
 	// GetVersion gets the version to be installed
 	GetVersion() string
+	// GetManifests gets the list of manifests to be installed
```

Comment:
Is "to be installed" valid? Maybe "which should ultimately be installed" to keep it a bit more declarative?